packages/ts/src/components/bullet-legend/index.ts

```diff
@@ -66,7 +69,13 @@ export class BulletLegend {
       .call(createBullets)
 
     legendItemsMerged.select<SVGElement>(`.${s.bullet}`)
-      .style('width', config.bulletSize)
+      .style('width', (d: BulletLegendItemInterface) => {
+        const colors = Array.isArray(d.color) ? d.color : [d.color]
+        const numColors = colors.length
+        const baseSize = parseInt(config.bulletSize || DEFAULT_BULLET_SIZE)
```

Comment:
What if `config.bulletSize` has units like `pt` or `em`? It's better to use the `to-px` package to convert